api/lib/controllers/users/actions.js

```diff
@@ -1,10 +1,23 @@
+const jwt = require('jsonwebtoken');
+const config = require('config');
+
 const UsersService = require('../../entities/users.service');
 const { sendActivateUserMail } = require('../auth/mail');
 const { appLogger } = require('../../services/logger');
 const { activateUserLink } = require('../auth/password');
 const { adminImportSchema, includableFields } = require('../../entities/users.dto');
 const { propsToPrismaInclude } = require('../utils');
 
+const secret = config.get('auth.secret');
+const cookie = config.get('auth.cookie');
+
+function generateToken(user) {
```

Comment:
This function already exist in [/api/lib/controllers/auth/auth.js](https://github.com/ezpaarse-project/ezmesure/blob/master/api/lib/controllers/auth/auth.js#L16). Maybe you need to put it in userServices to avoid duplicate code